docsite/components/toast.html

```diff
@@ -0,0 +1,107 @@
+                    <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28155%2C155%2C155%2C1%29&t=night-owl&wt=sharp&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2509%253Cbutton%2520class%253D%2522reset%2520padding-sm%2520btn-outline%2522%2520id%253D%2522toast-btn%2522%253EShow%2520Toast%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Cdiv%2520id%253D%2522toast%2522%253EHii%252C%2520I%2520am%2520toast.%2520I%2520will%2520be%2520gone%2520in%25203%2520seconds%253C%252Fdiv%253E%250A%2520%2520%2520%250A%2520%2520%2520%252F%252Fjs%250A%2520%2520%2520%2520var%2520toastBtn%2520%253D%2520document.querySelector%28%2522%2523toast-btn%2522%29%250A%2520%2520%2520%2520%2520%2520%2520%2520var%2520toast%2520%253D%2520document.querySelector%28%2522%2523toast%2522%29%250A%2520%2520%2520%2520%2520%2520%2520%2520toastBtn.addEventListener%28%2522click%2522%252C%28%29%253D%253E%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520toast.classList.add%28%2522show%2522%29%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520setTimeout%28%28%29%253D%253E%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520toast.classList.remove%28%2522show%2522%29%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%257D%252C3000%29%250A%2520%2520%2520%2520%2520%2520%2520%2520%257D%29"
+ class="iframe-lg"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+                </div>
+                
+            </div>
+        
+    </div>
+    </main>
+    
+    <footer class="centered vertical-direction bg-secondary">
+        <button class="prev-page-btn btn-transparent btn rm-border font2 float"><a class="decor-none text-vibrant" href="rating.html"><i class="fa fa-arrow-left "></i> Rating </a></button>
+        <button class="next-page-btn btn-transparent btn rm-border font2 float"><a class="decor-none text-vibrant" href="grid.html">Grid <i class="fa fa-arrow-right "></i> </a></button>
+       
+
+        <h2 class=" mb-lg ">Build with <span class="text-primary">&lt;/&gt;</span> by Kajal Kumari</h2>
+        <ul class="style-list-none mb-lg ">
+            <li class="inline-block padding-xs"><a class="light-text" href="httlps://github.com/developers-codz">
+                <i class='fab fa-github fa-lg'></i>
+            </a>
+               </li>
+            <li class="inline-block padding-xs"><a class="light-text" href="https://www.linkedin.com/in/kajal-kumari-52bab41aa"><i class='fab fa-linkedin fa-lg'></i></a></li>
+            <li class="inline-block padding-xs"><a class="light-text" href="https://twitter.com/Kajal3310"><i class='fab fa-twitter fa-lg'></i></a></li>
+        </ul>
+        <small>© 2022 beadsUI <a class="decor-none light-text" href="https://github.com/developers-codz">Developers-codz</a> </small>
+        
+    </footer>
+    <script>
```

Comment:
I think you should use external js.

Reply:
As there was lots of file got created I put js there but yeah separated it. Thanks for reviewing.